syntax = "proto3";

package lgbsttracker.common;

enum ErrorCode {
    INTERNAL_ERROR = 0;
    TEMPORARILY_UNAVAILABLE = 1;
    ENDPOINT_NOT_FOUND = 2;
    PERMISSION_DENIED = 3;
    REQUEST_LIMIT_EXCEEDED = 4;
    BAD_REQUEST = 5;
    INVALID_PARAMETER_VALUE = 6;
    RESOURCE_DOES_NOT_EXIST = 7;
    INVALID_STATE = 8;
    RESOURCE_ALREADY_EXISTS = 9;
}